x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/index.ts

```diff
@@ -99,6 +99,8 @@ export interface ParamEditorProps<C> {
   data: DataPublicPluginStart;
 }
 
+export type TimeScalingMode = 'disabled' | 'mandatory' | 'optional';
```

Comment:
It looks like mandatory time scaling isn't handled in this PR: nothing is different in the UI if it's mandatory vs optional. Intentional?

Reply:
The "remove" button is not shown (implicitly checking the absence of `mandatory` by checking for `optional` in a branch of the code that ruled out undefined and `disabled` already)